core/math/geometry.cpp

```diff
@@ -33,6 +33,8 @@
 #include "core/print_string.h"
 #include "thirdparty/misc/clipper.hpp"
 #include "thirdparty/misc/triangulator.h"
+#define STB_RECT_PACK_IMPLEMENTATION
+#include "thirdparty/stb_rect_pack/stb_rect_pack.h"
```

Comment:
Nitpick as it's preexisting code, but please add a separation line between Godot headers (`"core/print_string.h"`) and thirdparty/system headers.